data/remnant missions.txt

```diff
@@ -495,3 +560,772 @@ mission "Remnant: Technology Available"
+				accept
+			label library
+			`	"Excellent! According to a friend of mine who is versed in history from the war, there were rumors that a grand library was being built in the Deep to house all of human knowledge. If it exists..." she stops when she notices you nodding with a knowing expression. "You have already been there? Well, you certainly get around, captain <last>. I suppose that makes things rather straight forward."`
+				accept
+	on stopover
+		payment -40
+		conversation
+			`After having traveled clear across the galaxy, you finally have arrived at the archives of human history. On the direction of the librarians you locate the gift shop and buy a "souvenir copy" of the archive. You idly wonder if you should buy a couple spare copies just in case, but decide against it.`
+	
+	on complete
+		conversation
+			`Having acquired a copy of the archive, you gently set <ship> down on Viminal hoping that this information is helpful. The port seems unusually busy, but the comm channels are quiet beyond the bare minimum to keep traffic organized.`
+			`	Before you have even taken the time to shutdown the engines Tali is halfway up the side of your ship securing a refueling hose and charging cable. As you reach for the comlink she scrambles up to the bridge windows and quickly rattles off a series of signs at you. All you can make out is "Keep it hot, incoming!" Before you can frame a reply she has raced across your wing and leapt to the Albatross parked next to you.`
+			`	Looking around more carefully, you note that every ship in the port is idling with fuel lines still connected. If this were some Republic port the safety inspectors would be having a nightmare. One Starling is getting holes in its wings patched even as the crew appear to be running prep-for-flight checklists. It also looks like every ship has a guard. Something tells you that right now would be a bad time to take a walk around town.`
+
+
+
+mission "Remnant: Salvage 2"
+	landing
+	source "Viminal"
+	name "Defend <planet>"
+	description "Assist in defending <planet> from a Korath raid, then return to the planet."
+	to offer
+		has "Remnant: Salvage 1: done"
+	on offer
+		conversation
+			`In the flurry of activity and tense waiting for something, it is almost a relief when the planetary defense sirens start. As you watch, the method to their chaos is suddenly evident. Each guard or port hand who had been standing around reaches up and pulls the quick release cable attached to the fuel hose, then race to get themselves and the hoses back out of the way. Mechanics operating overhead slap the buttons on their harnesses to be whisked up to the ceiling of the dome at breakneck speeds. Within seconds the first ships are out of the dome and the rest follow closely behind.`
```

Comment:
"Race to get" should match with "each guard or port hand" in number - that is, singular.

Reply:
Derpy cut most of that section out, so it is now fixed. Thanks for pointing it out, though.